src/redux/countriesSlice/countriesSlice.js

```diff
@@ -0,0 +1,111 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
```

Comment:
- [ ] Kindly add some tests for your redux code, in this way, you will improve your code stability 💯

Reply:
Thank you for @BregornOriginal  your valueable comments I have done it!